deploy/examples/external/common-external.yaml

```
@@ -0,0 +1,83 @@
+###################################################################################################################
+# Create the common resources that are necessary to start an external Ceph cluster in same namespace
+# These resources can be created after an operator that is already running but assumes common.yaml has been injected
+# The samples all assume that your existing operator running "rook-ceph" namespace will also watch and have permissions
+# to interact with an external cluster configured in "rook-ceph-external" cluster.
+#
+# kubectl create -f ../../examples/crds.yaml -f ../../examples/common.yaml -f ../../examples/operator.yaml
+#
+# If there is no cluster managed by the current Rook Operator
+# you can simply replace all occurrence of rook-ceph-external with rook-ceph
+#
+# And remove the following code:
+#
+#   apiVersion: v1
+#   kind: Namespace
+#   metadata:
+#     name: rook-ceph-external
+#
+# Then kubectl create -f common-external.yaml -f cluster-external.yaml
+###################################################################################################################
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rook-ceph-external # namespace:cluster
```

Similar changes needed throughout this file
```suggestion
  name: rook-ceph # namespace:cluster
```

---

Yes, I had the same understanding, at the beginning I changed the namespace to `rook-ceph`. But later after rounds of review, I thought you wants to keep it same as before and will change it later. Basically I misunderstood and changed them back to original. But thanks for all the clarification. I am updating them asap.